src/webui/www/public/index.html

```diff
@@ -2,6 +2,14 @@
 <html lang="${LANG}">
 
 <head>
+    <script>
```

Comment:
Please move it to the next line of `</noscript>`.
The same for private/index.html.